README.md

```diff
@@ -26,7 +26,7 @@ This requires a Hypixel API key. You may obtain one by logging onto `hypixel.net
 const LilyWeight = require("lilyweight");
 
 // or import from Ecmascript Module
-import LilyWeight from "lilyweight"; 
+import {LilyWeight} from "lilyweight";
```

Comment:
I'm pretty sure those curly braces shouldn't be here. It should be a default export.

Reply:
Hmm. I can double-check it again, but when I tested imports I had to use the curly brace because without them it was only importing the type. In that scenario, I believe you would have to do something along these lines.
```typescript
import LilyWeight from "lilyweight"

const lily = new LilyWeight.LilyWeight(your key here)
```

Reply:
Try making the class a default export in your type declarations.
```ts
    export default class LilyWeight {
```

Reply:
Okay, the most recent code uses a default export for the class.